server/pkg/thread/builder_test.go

```diff
@@ -55,49 +56,19 @@ func TestBuilder_Build(t *testing.T) {
 }
 
 func TestBuilder_MustBuild(t *testing.T) {
-	var thid ID = NewID()
+	thid := NewID()
+	wid := NewWorkspaceID()
 	c := []*Comment{}
+	got := New().
+		ID(thid).
+		Comments(c).
+		Workspace(wid).
+		MustBuild()
+	want := lo.Must(New().
+		ID(thid).
+		Comments(c).Workspace(wid).Build())
 
-	tests := Tests{
-		{
-			name: "Valid thread",
-			input: Input{
-				id:       thid,
-				comments: c,
-			},
-			want: &Thread{
-				id:       thid,
-				comments: c,
-			},
-		},
-		{
-			name: "fail: Invalid Id",
-			input: Input{
-				id:       ID{},
-				comments: c,
-			},
-			err: ErrInvalidID,
-		},
-	}
-
-	for _, tt := range tests {
-		tt := tt
-		t.Run(tt.name, func(t *testing.T) {
-			t.Parallel()
-			build := func() *Thread {
-				t.Helper()
-				return New().
-					ID(tt.input.id).
-					Comments(tt.input.comments).
-					MustBuild()
-			}
-			if tt.err != nil {
-				assert.PanicsWithValue(t, tt.err, func() { _ = build() })
-			} else {
-				assert.Equal(t, tt.want, build())
-			}
-		})
-	}
+	assert.Equal(t, want, got)
```

Comment:
Add an abnormal test case that Must method panics.

To get an idea of how many test cases you need, consider increasing code coverage. That means eliminating as many untested lines as possible!

Reply:
Done!